front/app/hooks/useEvents.ts

```diff
@@ -3,7 +3,7 @@ import { isNilOrError } from 'utils/helperUtils';
 import { Observable, of } from 'rxjs';
 import { IEventData, IEvents, eventsStream } from 'services/events';
 
-export default function useProject(projectId: string | null | undefined) {
+export default function useProject(projectId?: string | null) {
```

Comment:
Super nitpicky but probably makes more sense to export this as `useEvents` right?

Reply:
Oops, I did change this but forgot to push, and that's not nitpicky at all 👍 Good that you point it out